pytorch_lightning/accelerators/accelerator.py

```diff
@@ -224,29 +223,23 @@ def predict(self, args: List[Union[Any, int]]) -> _STEP_OUTPUT_TYPE:
         with self.precision_plugin.predict_context(), self.training_type_plugin.predict_context():
             return self.training_type_plugin.predict(*args)
 
-    def training_step_end(
-        self, output: _STEP_OUTPUT_TYPE
-    ) -> _STEP_OUTPUT_TYPE:
+    def training_step_end(self, output: _STEP_OUTPUT_TYPE) -> _STEP_OUTPUT_TYPE:
```

Comment:
Are those formatting changes really necessary in this PR? they shouldn't be touched at all?

Reply:
Well, since will drop formating checks from particular PRs, I don't have many other options how to keep codebase healthy